src/main/java/hudson/plugins/parameterizedtrigger/BlockableBuildTriggerConfig.java

```diff
@@ -24,19 +26,24 @@
 
 /**
  * {@link BuildTriggerConfig} that supports blocking of the execution.
+ * 
  * @author Kohsuke Kawaguchi
  */
 public class BlockableBuildTriggerConfig extends BuildTriggerConfig {
     private final BlockingBehaviour block;
+    // Field is unused, but would change public API if it were deleted
+    @SuppressFBWarnings(value = "UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD", justification = "Part of the public API")
     public boolean buildAllNodesWithLabel;
 
-    public BlockableBuildTriggerConfig(String projects, BlockingBehaviour block, List<AbstractBuildParameters> configs) {
+    public BlockableBuildTriggerConfig(String projects, BlockingBehaviour block,
+            List<AbstractBuildParameters> configs) {
         super(projects, ResultCondition.ALWAYS, false, configs);
         this.block = block;
     }
 
     @DataBoundConstructor
-    public BlockableBuildTriggerConfig(String projects, BlockingBehaviour block, List<AbstractBuildParameterFactory> configFactories,List<AbstractBuildParameters> configs) {
+    public BlockableBuildTriggerConfig(String projects, BlockingBehaviour block,
+            List<AbstractBuildParameterFactory> configFactories, List<AbstractBuildParameters> configs) {
```

Comment:
By the way please avoid gratuitous diff hunks, to make reviewing real changes easier.

Reply:
Yes sure, sorry about that. I'll undo the formatting changes. 
My VS Code did these automatically, I'll keep this in mind from now on.

Reply:
Yeah some editors default to just reformatting everything on save. Fine for Go since everyone uses `gofmt` to begin with, but bad for diffs in other languages.